Sources/PhotoCaptureViewController/CaptureManager.swift

```diff
@@ -278,6 +294,40 @@ class CaptureManager: NSObject {
     }
 }
 
+// MARK: - AVCapturePhotoCaptureDelegate
+
+extension CaptureManager: AVCapturePhotoCaptureDelegate {
+    func photoOutput(_ output: AVCapturePhotoOutput, didFinishProcessingPhoto photoSampleBuffer: CMSampleBuffer?, previewPhoto previewPhotoSampleBuffer: CMSampleBuffer?, resolvedSettings: AVCaptureResolvedPhotoSettings, bracketSettings: AVCaptureBracketedStillImageSettings?, error: Error?) {
+
+        // We either call the delegate or the completion block not both.
+        if didCaptureImageCompletion != nil && delegate != nil {
+            didCaptureImageCompletion = nil
+        }
+
+        guard error == nil else {
+            NSLog("Failed capturing still images: \(String(describing: error))")
```

Comment:
Not sure how useful it is, but we could also add a delegate method to report back about errors.

Reply:
I agree 👍